src/prefect_server/services/loop_service.py

```diff
@@ -40,21 +44,19 @@ def __init__(self, loop_seconds: Union[float, int] = None):
         self.loop_seconds = float(loop_seconds)
         self.name = type(self).__name__
         self.logger = utilities.logging.get_logger(self.name)
+        self._stop_running = False
```

Comment:
we already have an `_is_running` attribute that you should probably reuse instead